src/plugins/intel_gpu/src/kernel_selector/kernels/permute/permute_kernel_f_y_axes.cpp

```diff
@@ -212,6 +212,15 @@ bool PermuteKernel_f_y_axes::Validate(const Params& p) const {
     };
 
     const auto& params = dynamic_cast<const permute_params&>(p);
+    // permute_kernel_f_y_axes doesn't support dynamic shape.
+    // So it needs to check if fused ops have dynamic shape.
+    for (auto& fused_op : params.fused_ops) {
+        for (auto tensor : fused_op.tensors) {
+            if (tensor.is_dynamic())
+                return false;
+        }
+    }
+
```

Comment:
This should be checked by EnabledynamicshapesSupport(). 
Seems that it is not set for this kernel, so could you please check why dynamic shape case is not filtered by EnableDynamicShapesSupport?

Reply:
At the time of filtering by _EnableDynamicShapesSupport()_, the problematic permute node is not filtered because it has a static shape. At the time of building after _prepare_primitive_fusing_, the node that receives the dynamic shape as input is fused to the permute node. Therefore, I think _Validate()_ should check whether the fused node has a dynamic shape.

Reply:
Changed the existing fix that only fixes permute to make it applicable to all other kernels as well.